crm_claim/views/crm_claim_stage_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="crm_claim_stage_tree" model="ir.ui.view">
+        <field name="name">crm.claim.stage.tree</field>
+        <field name="model">crm.claim.stage</field>
+        <field name="arch" type="xml">
+            <tree string="Claim Stages">
+                <field name="sequence"/>
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="crm_claim_stage_form" model="ir.ui.view">
+        <field name="name">crm.claim.stage.form</field>
+        <field name="model">crm.claim.stage</field>
+        <field name="arch" type="xml">
+            <form string="Claim Stage">
+                <group col="4">
+                    <field name="name"/>
+                    <field name="case_default"/>
```

Comment:
Isn't it a little bit hard to know wich team a stage belongs to?

Reply:
You can't know which team a stage belongs to, because stage isn't a field of crm.team model.

Case_default is used to assign a default stage based on team to crm_claim

Reply:
@cubells please answer this

Reply:
@chienandalu I am agree with your point. We have to set <code>case_default</code> to <code>True</code>. So we can visible <code>Teams</code> field into Stage. Afterwards above domain will works fine. I have created PR (https://github.com/OCA/crm/pull/178) for that.